dpnp/dpnp_iface_bitwise.py

```diff
@@ -132,20 +105,8 @@ def bitwise_or(x1, x2):
     logical_or, bitwise_and, bitwise_xor, binary_repr
 
     """
-
-    def numpy_bitwise_or(x1, x2):
-        x1 = dpnp.asnumpy(x1) if isinstance(x1, dparray) else x1
-        x2 = dpnp.asnumpy(x2) if isinstance(x2, dparray) else x2
-        return numpy.bitwise_or(x1, x2)
-
-    if use_origin_backend(x1):
-        return numpy_bitwise_or(x1, x2)
-
-    valid_types = (numpy.int64, numpy.int32, numpy.bool)
-    if use_dpnp_backend(x1, x2, valid_types):
-        return dpnp_bitwise_or(x1, x2)
-
-    return numpy_bitwise_or(x1, x2)
+    valid_dtypes = [numpy.int64, numpy.int32]
+    return check2call(numpy.bitwise_or, dpnp_bitwise_or, valid_dtypes, x1, x2, check_shapes=True, check_dtypes=True)
 
 
 def bitwise_xor(x1, x2):
```

Comment:
checks for `[numpy.int64, numpy.int32]` can be used in the `x1.shape != x2.shape` way but with no throw exception.
if x1.dtype is in supported_types:
    return dpnp_bitwise_xor()

Reply:
@shssf what is better approach in case of different shapes, sizes or dtypes of input arrays? Do we need to raise exception with `checker_throw_value_error` or give a chance to `numpy` to handle this case by calling original (`numpy`) function? I would like to polish function `check2call` (or better name it `check_nd_call`) to handle these cases.

I mean something like that:
```python
if (x1.size != x2.size): 
    checker_throw_value_error("add", "size", x1.size, x2.size)
```
or
```python
if (x1.size != x2.size):
    call_origin(numpy.add, x1, x2, **kwargs)
```

P.S. what do you think the function `check2call` makes sense or not?

Reply:
I think the best practice to let numpy decide.
We need to do our calculation for only cases where it is profitable. All other must go to fallback.
So, if we have no support for some `sizes` - it must go to fallback (numpy)

Reply:
I'm not sure but it looks like `check2call` is duplication of existed functions (similar) in other modules.